app/views/practices/instructions.html.erb

```diff
@@ -19,30 +19,51 @@
         <section class="margin-bottom-1">
           <%= render partial: "shared/messages", locals: {small_text: false} %>
           <h1 class="font-sans-2xl margin-bottom-205 line-height-sans-205">Create your practice page</h1>
-          <h2 class="margin-0 text-normal">Please follow these instructions to complete each of the following sections for your practice page.</h2>
+          <h2 class="margin-0 text-normal line-height-36">Please follow these instructions to complete each of the following sections for your practice page.</h2>
         </section>
-
-        <section>
-          <div class="padding-y-1">
+        <br/>
+          <section class="desktop:grid-col-8">
+            <h2 class="margin-0 text-bold">What you'll need</h2>
+              <div class="">
+                <ul class="line-height-sans-6 font-sans-md instructions-list">
+                  <li>To create a thorough page, you’ll be asked to share data, images, video, instructions, files, and stories related to your practice.</li>
+                  <li>All uploaded files should be displayed in a common file format (.jpg, .jpeg, .png, or .pdf) that is ready to be shown to the public. Images must be less than 32MB.</li>
+                </ul>
+              </div>
+          </section>
+          <section class="desktop:grid-col-8">
+          <h2 class="margin-0 text-bold">PII and PHI policy</h2>
+          <div class="">
             <ul class="line-height-sans-6 font-sans-md instructions-list">
-              <li>Do not enter Personally Identifying Information (PII) or Personal Health Information (PHI) for any individual, Veteran, or patient into the Diffusion Marketplace.</li>
-              <li>All uploaded files (research, data, examples, illustrations, etc.) should be displayed in a general format (e.g. PNG, JPG, PDF) that is ready to be shown to the public.</li>
-              <li>Ensure your work is accurate, formatted consistently, and is complete before publishing. Use sentence case and spell out acronyms. Your published responses will be displayed ‘as-is’ on the Diffusion Marketplace.</li>
-              <li>You will be able to edit your practice once it’s published.</li>
+              <li>Do not enter Personally Identifiable Information (PII) or Protected Health Information (PHI) for any individual, Veteran, or patient.</li>
+              <li><%= link_to "Learn more", pii_phi_information_path %> about what constitutes PII and PHI.</li>
+              <li>If you want to upload an image that features a Veteran, or text that includes a Veteran’s name, you must have a waiver. Waivers must be filled out with the ‘External to VA’ check box selected.</li>
```

Comment:
in the designs it looks "a waiver" has a link to something (I'm assuming a waiver form?) did we want to add that here?

Reply:
or are we not because of the different types of consent forms available?

Reply:
yeah, we just showed this to becca. her original design didn't accomdate the two forms, so we went with the updated design which is in the "Privacy Policy" for the new Practice Editor instructions page

Reply:
actually we'll probably have to update this

Reply:
to the new text